src/components/ExpenseForm/ExpenseForm.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react'
+import { DataService } from '../../services/DataService'
+import DatePicker from 'react-datepicker'
+import { expensesUrl } from '../../static/apiconfig'
+
+import "react-datepicker/dist/react-datepicker.css"
+
+import * as Styled from './style';
+
+const  ExpenseForm = () => {
+  const [cost, setCost] = useState('')
```

Comment:
shouldn't setCost and setCategory be function definitions as defaults rather than empty strings?